### PR TITLE
Fixes to "High School Sweethearts" and GitHub actions

### PR DESCRIPTION
**Pull request by mk-mxp**
Hi all,

thanks a lot for this great tool and learning experience. I'm happy to provide something back.

- Fixed typo in the name of "High School Sweethearts"
- Added a missing test for whitespace removal in the first task
- As I had GitHub actions with warnings, I fixed the PHP version setting of the unit testing action

I hope these break nothing out of the scope of this repo.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
       {
         "uuid": "d6b00994-36b1-4d2c-9a7f-d55f3cbdf4de",
         "slug": "sweethearts",
-        "name": "Highschool Sweetheats",
+        "name": "Highschool Sweethearts",
```
Comment (kotp) [inline config.json:41]: Good catch.  Does not appear to be available in |problem-specifications" upstream, but I wonder if other exercise tracks have this same issue.

**Review by neenjaw — APPROVED** [review]
